client/src/components/Comment/Comment.css

```diff
@@ -0,0 +1,67 @@
+.Comment_body {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+  width: 50rem;
+  height: auto;
+  background-color: var(--gray-dark);
+  font-size: 14px;
+  font-family: var(--paragraph-font);
+  padding: 1.2rem;
+}
+
+.Comment_h4 {
```

Comment:
Can you come up with a name for this, rather than using the tag name? We want the name to be semantic